grug/math/src/conversions.rs

```diff
@@ -1,22 +1,29 @@
-use crate::{Dec, FixedPoint, Int, MathError, MathResult, Number};
+use {
+    crate::{Dec, FixedPoint, Int, MathError, MathResult, Number, NumberConst},
+    std::cmp::Ordering,
+};
 
 // -------------------------------- int -> dec ---------------------------------
 
-impl<U> Int<U>
+pub trait IntoDec<U, const S: u32> {
+    fn checked_into_dec(self) -> MathResult<Dec<U, S>>;
+}
```

Comment:
Does this have to be a trait? Imo we either use trait for conversions everywhere, or nowhere... for consistency.